app/views/games/_table.html.erb

```diff
@@ -10,20 +10,19 @@
 <% else %>
   <table class="game-table table table-bordered table-striped">
     <tr>
-      <th>Name</th>
-      <% if show_console %>
-        <th>Console</th>
```

Comment:
Are we not showing the console column anymore? That can be super useful for games that came out on multiple systems (like Metal Gear Solid)